packages/recipe/src/components/EzLabel/Documentation/Stories/Default.stories.tsx

```diff
@@ -1,8 +1,58 @@
 import React from 'react';
 import {type Meta, type StoryObj} from '@storybook/react';
-import EzLabel from '../../EzLabel';
+import dedent from 'ts-dedent';
+import EzLabel, {Props} from '../../EzLabel';
 
 const meta: Meta<typeof EzLabel> = {
+  argTypes: {
+    as: {
+      control: {type: 'select'},
+      description: 'The element type for the label.',
+      options: ['label', 'legend', 'div'],
+      table: {
+        defaultValue: {summary: 'label'},
+        type: {summary: 'label | legend | div'},
+      },
+      type: {name: 'string', required: false},
+    },
+    htmlFor: {
+      control: {type: 'text'},
+      description:
+        'Associates a label with an input, and should be the same as the id of the input. Alternatively, nest the input element directly inside the label.',
+      table: {
+        type: {summary: 'string'},
+      },
+      type: {name: 'string', required: false},
+    },
+    use: {
+      control: {type: 'select'},
+      description: 'The style of the label for the selected size.',
```

Comment:
```suggestion
      description: 'The use or variant of the label.',
```
To be consistent with descriptions of other components with this prop.